OracleJava/java-8/windows/server-jre-8u112-windows-x64.tar.gz.download

```diff
@@ -0,0 +1,5 @@
+# Download the Server JRE (which includes necessary JDK bits for backend Java based solutions)
+#
+#  - http://www.oracle.com/technetwork/java/javase/downloads/server-jre8-downloads-2133154.html
+#
+e2fa735e6758e7c6bda43ba7cbaffe9c  server-jre-8u112-linux-x64.tar.gz
```

Comment:
I haven't adjusted the checksum here as I don't know which sha type I should use or where to fetch the prebuilt checksum.